libbeat/idxmgmt/idxmgmt_test.go

```diff
@@ -86,81 +84,6 @@ func TestDefaultSupport_Enabled(t *testing.T) {
 	}
 }
 
-func TestDefaultSupport_TemplateConfig(t *testing.T) {
```

Comment:
Did we move these tests somewhere else?

Reply:
Nope, simply removed them, as there is no `TemplateConfig()` method any more.

Reply:
Removing tests always makes me unseasy :). Would it make sense to reimplement some of them in another location?

Reply:
Added them back considering the new logic